src/plugins/visualizations/public/vis_types/base_vis_type.ts

```diff
@@ -44,7 +44,7 @@ interface CommonBaseVisTypeOptions {
   useCustomNoDataScreen?: boolean;
   inspectorAdapters?: Adapters | (() => Adapters);
   isDeprecated?: boolean;
-  getDeprecationMessage?: (vis: Vis) => ReactElement<{}>;
+  getInfoMessage?: (vis: Vis) => ReactElement<{}>;
```

Comment:
@timroes @stratoula @sulemanof  Probably we can use `getDeprecationMessage` to archive out goal. 
But name should be more generic. 

One more point if we decide to go with it, in this case we should parse spec twice =( But probably it's not a problem

Reply:
I am fine with this approach 🙂